### ISSUE TITLE
Win 10 Start Menu doesn't work in the latest Windows 11 Insider Preview Build 23595 (Dev Channel)

### ISSUE THREAD

**Issue by hoiyuyiu** [valinet/ExplorerPatcher#2548]
i can't change my start menu style after update to Build 23595, and it worked well in last dev 23590.

**Comment by hoiyuyiu** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/61400426/4990bbba-bfb7-4c1e-8a90-69f6bb0f8307)

**Comment by pyrates999** [comment]
Canary builds are not supported because the code is not final and can change with each new release.

**Comment by hoiyuyiu** [comment]
> Please try the latest pre-release since you are running a dev build of windows: https://github.com/valinet/ExplorerPatcher/releases请尝试最新的预发布，因为您正在运行Windows的开发版本：https://github.com/valinet/ExplorerPatcher/releases

I just installed, and my problem has been solved.

My EP was updated to 22621.2506.60.1 via auto update, but idk it isn't the latest pre-release, so i reported my problem. 

And I found that there are 3 different 22621.2506.60.1 (and i didn't realize that they are not the same until i found the different letters after "_", which is not clearly shown at the title), i think it's the reason that i can't auto update to the latest one even if I CHECK the "Receive pre-release version..." option.
